packages/core/src/login/oidc/IOidcOptions.ts

```diff
@@ -33,28 +33,31 @@ import { IClient } from "./IClient";
 /**
  * @hidden
  */
-export interface ICoreOidcOptions {
+export interface IOidcOptions {
+  /**
+   * The URL of the Solid Identity Provider.
+   */
   issuer: string;
+  /**
+   * The openid-configuration of the issuer.
+   */
   issuerConfiguration: IIssuerConfig;
   client: IClient;
   sessionId: string;
   refreshToken?: string;
-}
-
-/**
- * @hidden
- */
-export interface IAccessTokenOidcOptions extends ICoreOidcOptions {
+  /**
+   * Specify wether the Solid Identity Provider may or may interact with the user.
```

Comment:
Ditto to the comment above - same typos here too...

Reply:
2f7f4b921cbb20661cc1adccba736312f7e587da